Library/Homebrew/cmd/search.rb

```diff
@@ -157,7 +157,7 @@ def search_formulae(rx)
       if aliases.include?(name) && results.include?(canonical_full_name)
         next
       elsif (HOMEBREW_CELLAR/canonical_name).directory?
-        "#{name} (installed)"
+        pretty_installed(name)
```

Comment:
The above no longer applies, obviously. As for the use of `search_formulae` in `brew install`, I guess it's better to sort that out in #45707.